boards/nuvoton/numaker_m2l31ki/board.cmake

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(pyocd "--target=m2l31kidae")
+board_runner_args(nulink "-f")
+
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/nulink.board.cmake)
```

Comment:
hey just noticed this one:

```console
$ west flash -r nulink
-- west flash: using runner nulink
FATAL ERROR: unknown runner "nulink"
```

Are you going to upstream this?

Reply:
So far, no plan and I remove this proprietary nulink runner.
Thanks & update commit 48edae5b230c

Reply:
Ok thanks for clarifying, the whole `boards/common/nulink.board.cmake` should probably go as well then. That said, I think it should be acceptable to keep it as long as you contribute the west runner script as well (the file in `scripts/west_commands/runners`, even if the binary it self is not distributed with the project. At least that's how nrfutil and nrfjprog works already (and probably more).